appmgr/src/net/wifi.rs

```diff
@@ -517,6 +517,15 @@ pub fn country_code_parse(code: &str, _matches: &ArgMatches<'_>) -> Result<Count
     )))
 }
 
+pub async fn synchronize_wpa_supplicant_conf(main_datadir: &std::path::Path) -> Result<(), Error> {
+    let source = main_datadir.join("wpa_supplicant.conf");
+    if tokio::fs::metadata(&source).await.is_err() {
+        tokio::fs::write(&source, include_str!("wpa_supplicant.conf.template")).await?;
+    }
+    tokio::fs::symlink(&source, "/etc/wpa_supplicant.conf").await?;
```

Comment:
this will error if /etc/wpa_supplicant.conf already exists. add a line before to delete it if it does

Reply:
Is that actually what we want? In theory it shouldn't ever exist prior to this.

Reply:
the less we rely on a very specific host configuration, the better. especially when this gets ported to being a dpkg

Reply:
I just am not sure that silently deleting and overwriting it is the correct behavior here. Seems like erroring or abdicating WiFi responsibility is the right call in that scenario.